t/apicast-policy-upstream.t

```diff
@@ -0,0 +1,125 @@
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+run_tests();
+
+__DATA__
+
```

Comment:
@mikz  I also wanted to check a case with a single rule that does not match. However, I don't know what's the best way to test that. The request would try to go to an invalid upstream. Not sure what would be the best way to mock it.